src/v2/view-builder/internals/BaseForm.js

```diff
@@ -130,16 +131,19 @@ export default Form.extend({
     if (Array.isArray(messages.value) && !(options instanceof View)) {
       this.add('<div class="ion-messages-container"></div>', errContainer);
       messages.value.forEach(obj => {
-        if(!obj?.class || obj.class === 'INFO') {
+        const {
+          class: msgClass = '',
+          title = '',
+          message,
+        } = obj;
+        if(!msgClass || msgClass === INFO_MESSAGE_CLASS) {
           // add message as plain text
-          this.add(`<p>${obj.message}</p>`, '.ion-messages-container');
+          this.add(`<p>${message}</p>`, '.ion-messages-container');
         } else {
-          // add error callout with custom options
-          options = Object.assign(_.pick(obj, 'message', 'class'), options);
```

Comment:
can you check after removing this.. custom error callouts still work ?

Reply:
there are few views that provides `options` as arguments when they trigger a call to `showMessages`  when `showCustomFormErrorCallout` is not null..

Reply:
How do I test custom error callouts?